ksql-common/src/main/java/io/confluent/ksql/util/timestamp/TimestampExtractionPolicyFactory.java

```diff
@@ -31,9 +37,23 @@ public static TimestampExtractionPolicy create(
       final KsqlSchema schema,
       final Optional<String> timestampColumnName,
       final Optional<String> timestampFormat
+  ) {
```

Comment:
This was used only in one place, `LogicalPlanner`. Why not remove this method and just use the next one?

Reply:
I'll do that, thanks.

Reply:
Done